### ISSUE TITLE
CogVideoXPipeline.__call__() got an unexpected keyword argument 'context_schedule'

### ISSUE THREAD

**Issue by Twelve615** [kijai/ComfyUI-CogVideoXWrapper#135]
---
system version: Linux featurize 5.15.0-105-generic #115-Ubuntu SMP Mon Apr 15 09:52:04 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
python version: 3.11.5
PyTorch: 2.4.1+cu121
CUDA: 12.6
---

**I don't know what went wrong. I hope this information can help you.**

## Workflow
[CogVideoMZ-basic.json](https://github.com/user-attachments/files/17276494/CogVideoMZ-basic.json)

## Detail
```log
[2024-10-07 09:41] got prompt
[2024-10-07 09:41] The config attributes {'snr_shift_scale': 1.0} were passed to DPMSolverMultistepScheduler, but are not expected and will be ignored. Please verify your scheduler_config.json configuration file.
[2024-10-07 09:41] !!! Exception during processing !!! CogVideoXPipeline.__call__() got an unexpected keyword argument 'context_schedule'
[2024-10-07 09:41] Traceback (most recent call last):
  File "/home/featurize/work/app/comfyui/ComfyUI/execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/featurize/work/app/comfyui/ComfyUI/execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/featurize/work/app/comfyui/ComfyUI/execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)
  File "/home/featurize/work/app/comfyui/ComfyUI/execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/featurize/work/app/comfyui/ComfyUI/custom_nodes/ComfyUI-CogVideoXWrapper/nodes.py", line 903, in process
    latents = pipeline["pipe"](
              ^^^^^^^^^^^^^^^^^
  File "/home/featurize/work/app/comfyui/env/lib/python3.11/site-packages/torch/utils/_contextlib.py", line 116, in decorate_context
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
TypeError: CogVideoXPipeline.__call__() got an unexpected keyword argument 'context_schedule'

[2024-10-07 09:41] Prompt executed in 1.64 seconds
```
## pip list
```log
Package                           Version
--------------------------------- ------------------
absl-py                           2.1.0
accelerate                        0.34.2
addict                            2.4.0
aggdraw                           1.3.19
aiohappyeyeballs                  2.4.0
aiohttp                           3.10.5
aioresponses                      0.7.6
aiosignal                         1.3.1
albucore                          0.0.16
albumentations                    1.4.15
annotated-types                   0.7.0
anthropic                         0.34.2
antlr4-python3-runtime            4.9.3
anyio                             4.5.0
APScheduler                       3.10.1
argostranslate                    1.9.6
arrow                             1.3.0
attrs                             24.2.0
awscli                            1.33.44
backoff                           2.2.1
beautifulsoup4                    4.12.3
binaryornot                       0.4.4
bitsandbytes                      0.43.3
blend-modes                       2.1.0
blind-watermark                   0.4.4
boto3                             1.34.123
botocore                          1.34.162
build                             1.2.2
cachetools                        5.5.0
certifi                           2024.8.30
cffi                              1.17.1
chardet                           5.2.0
charset-normalizer                3.3.2
click                             8.1.7
clip-interrogator                 0.6.0
cmake                             3.30.3
color-matcher                     0.5.0
colorama                          0.4.4
coloredlogs                       15.0.1
colorlog                          6.8.2
colour-science                    0.4.4
contourpy                         1.3.0
cookiecutter                      2.6.0
cryptography                      43.0.1
cssselect2                        0.7.0
cstr                              0.1.0
ctranslate2                       4.4.0
cycler                            0.12.1
Cython                            3.0.0
dataclasses-json                  0.6.7
ddt                               1.7.2
deep-translator                   1.11.4
deepdiff                          8.0.1
defusedxml                        0.7.1
Deprecated                        1.2.14
diffusers                         0.30.3
dill                              0.3.8
distro                            1.9.0
dlib                              19.24.6
docker                            6.1.3
docker-pycreds                    0.4.0
docopt                            0.6.2
docutils                          0.16
easydict                          1.13
einops                            0.8.0
embreex                           2.17.7.post5
eval_type_backport                0.2.0
facexlib                          0.3.0
fairscale                         0.4.13
fastapi                           0.110.3
ffmpy                             0.3.0
filelock                          3.16.1
filterpy                          1.4.5
flatbuffers                       24.3.25
flet                              0.24.1
flet-core                         0.24.1
flet-runtime                      0.24.1
fonttools                         4.53.1
frozenlist                        1.4.1
fsspec                            2024.9.0
ftfy                              6.2.3
fvcore                            0.1.5.post20221221
gdown                             5.2.0
gguf                              0.10.0
gitdb                             4.0.11
GitPython                         3.1.43
google                            3.0.0
google-ai-generativelanguage      0.6.9
google-api-core                   2.20.0
google-api-python-client          2.146.0
google-auth                       2.35.0
google-auth-httplib2              0.2.0
google-cloud-core                 2.4.1
google-cloud-storage              2.18.2
google-crc32c                     1.6.0
google-generativeai               0.8.1
google-resumable-media            2.7.2
googleapis-common-protos          1.65.0
googletrans-py                    4.0.0
GPUtil                            1.4.0
grpcio                            1.66.1
grpcio-status                     1.62.3
h11                               0.14.0
h2                                4.1.0
hpack                             4.0.0
httpcore                          1.0.5
httplib2                          0.22.0
httptools                         0.6.1
httpx                             0.27.2
huggingface-hub                   0.25.0
humanfriendly                     10.0
hydra-core                        1.3.2
hyperframe                        6.0.1
idna                              3.10
imageio                           2.35.1
imageio-ffmpeg                    0.5.1
img2texture                       1.0.6
importlib_metadata                8.5.0
inference-cli                     0.19.0
inference-gpu                     0.19.0
iniconfig                         2.0.0
insightface                       0.7.3
iopath                            0.1.10
jax                               0.4.33
jaxlib                            0.4.33
Jinja2                            3.1.4
jiter                             0.5.0
jmespath                          1.0.1
joblib                            1.4.2
json_repair                       0.29.2
jsonschema                        4.23.0
jsonschema-specifications         2023.12.1
kiwisolver                        1.4.7
kornia                            0.7.3
kornia_rs                         0.1.5
lark-parser                       0.12.0
lazy_loader                       0.4
llvmlite                          0.43.0
loguru                            0.7.2
loralib                           0.1.2
lpips                             0.1.4
lxml                              5.3.0
manifold3d                        2.5.1
mapbox_earcut                     1.0.2
markdown-it-py                    3.0.0
MarkupSafe                        2.1.5
marshmallow                       3.22.0
matplotlib                        3.8.0
matrix-client                     0.4.0
mdurl                             0.1.2
mediapipe                         0.10.15
ml_dtypes                         0.5.0
mpmath                            1.3.0
mss                               9.0.2
multidict                         6.1.0
mypy-extensions                   1.0.0
natsort                           8.4.0
networkx                          3.3
numba                             0.60.0
numexpr                           2.10.1
numpy                             1.26.4
nvidia-cublas-cu12                12.1.3.1
nvidia-cuda-cupti-cu12            12.1.105
nvidia-cuda-nvrtc-cu12            12.1.105
nvidia-cuda-runtime-cu12          12.1.105
nvidia-cudnn-cu12                 9.1.0.70
nvidia-cufft-cu12                 11.0.2.54
nvidia-curand-cu12                10.3.2.106
nvidia-cusolver-cu12              11.4.5.107
nvidia-cusparse-cu12              12.1.0.106
nvidia-nccl-cu12                  2.20.5
nvidia-nvjitlink-cu12             12.6.68
nvidia-nvtx-cu12                  12.1.105
oauthlib                          3.2.2
omegaconf                         2.3.0
onnx                              1.16.2
onnxruntime                       1.19.2
onnxruntime-gpu                   1.15.1
open_clip_torch                   2.26.1
openai                            1.46.1
opencv-contrib-python             4.10.0.84
opencv-python                     4.10.0.84
opencv-python-headless            4.10.0.84
opt-einsum                        3.3.0
orderly-set                       5.2.2
packaging                         24.1
pandas                            2.2.2
peft                              0.13.0
pendulum                          3.0.0
piexif                            1.1.3
pilgram                           1.2.1
pillow                            10.4.0
pip                               23.3.1
pip-tools                         7.4.1
pixeloe                           0.0.10
platformdirs                      4.3.6
pluggy                            1.5.0
pooch                             1.8.2
portalocker                       2.10.1
prettytable                       3.11.0
prometheus_client                 0.20.0
prometheus-fastapi-instrumentator 6.0.0
proto-plus                        1.24.0
protobuf                          4.25.5
psd-tools                         1.9.34
psutil                            6.0.0
PuLP                              2.9.0
py-cpuinfo                        9.0.0
pyasn1                            0.6.1
pyasn1_modules                    0.4.1
pybase64                          1.4.0
pycollada                         0.8
pycparser                         2.22
pydantic                          2.9.2
pydantic_core                     2.23.4
pydantic-settings                 2.5.2
pydot                             3.0.1
PyGithub                          2.4.0
Pygments                          2.18.0
PyJWT                             2.9.0
PyMatting                         1.1.12
PyNaCl                            1.5.0
pyOpenSSL                         24.2.1
pyparsing                         3.1.4
pypng                             0.20220715.0
pyproject_hooks                   1.1.0
PySocks                           1.7.1
pytest                            8.3.3
pytest-asyncio                    0.21.1
python-dateutil                   2.9.0.post0
python-dotenv                     1.0.1
python-slugify                    8.0.4
pytz                              2024.2
PyWavelets                        1.7.0
PyYAML                            6.0.2
pyzbar                            0.1.9
qrcode                            7.4.2
redis                             5.0.8
referencing                       0.35.1
regex                             2024.9.11
rembg                             2.0.59
repath                            0.9.0
reportlab                         4.2.4
requests                          2.31.0
requests-file                     2.1.0
requests-toolbelt                 1.0.0
requirements-parser               0.11.0
rf-clip                           1.1
rich                              13.5.2
rich-argparse                     1.5.2
rpds-py                           0.20.0
rsa                               4.7.2
Rtree                             1.3.0
s3transfer                        0.10.2
sacremoses                        0.0.53
safetensors                       0.4.5
scenedetect                       0.6.4
scikit-image                      0.24.0
scikit-learn                      1.5.2
scipy                             1.14.1
seaborn                           0.13.2
segment-anything                  1.0
Send2Trash                        1.8.3
sentencepiece                     0.2.0
sentry-sdk                        2.14.0
setproctitle                      1.3.3
setuptools                        72.1.0
shapely                           2.0.6
shellingham                       1.5.4
simpleeval                        0.9.13
six                               1.16.0
skypilot                          0.5.0
smmap                             5.0.1
sniffio                           1.3.1
socksio                           1.0.0
sounddevice                       0.5.0
soundfile                         0.12.1
soupsieve                         2.6
spandrel                          0.4.0
stanza                            1.1.1
starlette                         0.37.2
structlog                         24.4.0
supervision                       0.22.0
surrealist                        1.0.0
svg.path                          6.3
svglib                            1.5.1
sympy                             1.13.3
tabulate                          0.9.0
termcolor                         2.4.0
text-unidecode                    1.3
threadpoolctl                     3.5.0
tifffile                          2024.8.30
time-machine                      2.15.0
timm                              0.6.13
tinycss2                          1.3.0
tldextract                        5.1.2
tokenizers                        0.19.1
tomli                             2.0.1
torch                             2.4.1
torchaudio                        2.4.1
torchscale                        0.3.0
torchsde                          0.2.6
torchvision                       0.19.1
tqdm                              4.66.5
trampoline                        0.1.2
transformers                      4.44.2
transparent-background            1.3.2
trimesh                           4.4.9
triton                            3.0.0
typer                             0.9.0
typer-config                      1.4.0
types-python-dateutil             2.9.0.20240906
types-setuptools                  75.1.0.20240917
typing_extensions                 4.12.2
typing-inspect                    0.9.0
tzdata                            2024.1
tzlocal                           5.2
ultralytics                       8.2.98
ultralytics-thop                  2.0.6
uritemplate                       4.1.1
urllib3                           1.26.20
uvicorn                           0.30.6
uvloop                            0.20.0
vhacdx                            0.0.8.post1
vtracer                           0.6.11
wandb                             0.18.1
watchdog                          4.0.2
watchfiles                        0.24.0
wcwidth                           0.2.13
webencodings                      0.5.1
websocket-client                  1.8.0
websockets                        13.0.1
wget                              3.2
wheel                             0.41.2
wrapt                             1.16.0
xatlas                            0.0.9
xxhash                            3.5.0
yacs                              0.1.8
yapf                              0.40.2
yarl                              1.11.1
zipp                              3.20.2
```

**Comment by kijai** [comment]
> I noticed a closed 'issues' #46 It was updated and the problem disappeared but mine is still there; the version of comfyUI I'm using is [v0.2.2](https://github.com/comfyanonymous/ComfyUI/releases/tag/v0.2.2), the latest version used by CogVideoXWrapper

Can you try with one of my example workflows included in the examples folder instead?

**Comment by Twelve615** [comment]
> > I noticed a closed 'issues' #46 It was updated and the problem disappeared but mine is still there; the version of comfyUI I'm using is [v0.2.2](https://github.com/comfyanonymous/ComfyUI/releases/tag/v0.2.2), the latest version used by CogVideoXWrapper
> 
> Can you try with one of my example workflows included in the examples folder instead?

I used the example workflow in the examples folder and it ran successfully. This seems to be a problem with [CogVideoX-MZ](https://github.com/MinusZoneAI/ComfyUI-CogVideoX-MZ) when it comes to compatibility with your `CogVideo Sampler`!

Thank you for your reply, I will look for a solution again.
